pkg/infrastructure/services/infinispan.go

```diff
@@ -12,58 +12,167 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package infrastructure
+package services
 
 import (
-	"time"
-
 	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
 	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"time"
 )
 
 const (
-	// InfinispanSecretUsernameKey is the secret username key set in the linked secret
-	InfinispanSecretUsernameKey = "username"
-	// InfinispanSecretPasswordKey is the secret password key set in the linked secret
-	InfinispanSecretPasswordKey = "password"
-
-	infinispanEnvKeyUsername          string = "INFINISPAN_USERNAME"
-	infinispanEnvKeyPassword          string = "INFINISPAN_PASSWORD"
-	infinispanEnvKeyUseAuth           string = "INFINISPAN_USEAUTH"
-	infinispanEnvKeyAuthRealm         string = "INFINISPAN_AUTHREALM"
-	infinispanEnvKeySasl              string = "INFINISPAN_SASLMECHANISM"
-	infinispanEnvKeyCredSecret        string = "INFINISPAN_CREDENTIAL_SECRET"
-	infinispanEnvKeyServerList        string = "INFINISPAN_CLIENT_SERVER_LIST"
-	infinispanEnvKeyQuarkusServerList string = "QUARKUS_INFINISPAN_CLIENT_SERVER_LIST"
+	// vars for services that relies on infinispan image module, keeping for backward compatibility, should be removed:
+	// https://issues.redhat.com/browse/KOGITO-2046
+	// Deprecated
+	infinispanEnvKeyUsername string = "INFINISPAN_USERNAME"
+	// Deprecated
+	infinispanEnvKeyPassword string = "INFINISPAN_PASSWORD"
+	// Deprecated
+	infinispanEnvKeyUseAuth string = "INFINISPAN_USEAUTH"
+	// Deprecated
+	infinispanEnvKeyAuthRealm string = "INFINISPAN_AUTHREALM"
+	// Deprecated
+	infinispanEnvKeySasl string = "INFINISPAN_SASLMECHANISM"
+	// Deprecated
+	infinispanEnvKeyServerList string = "INFINISPAN_CLIENT_SERVER_LIST"
+
+	infinispanEnvKeyCredSecret string = "INFINISPAN_CREDENTIAL_SECRET"
+
+	// keys for infinispan vars definition
+
+	envVarInfinispanServerList    = "SERVER_LIST"
```

Comment:
The  variable values don't seem to be used anywhere, what about using https://yourbasic.org/golang/iota/ for them to hint that the value is not relevant?

Reply:
We could also add `iota`in our PR checks! Nice tip. Thanks. Will update the PR.

Reply:
See: https://github.com/kiegroup/kogito-cloud-operator/pull/326/files/85b88a74001015c2a214abbea24c3defc2e16ffb#diff-d39931a5e891cc3bca1e5251ba5ae3bfR64

And the IDE GoLand warn us about unused variables/usage values.

Reply:
Yes, by "not being used" I meant that the value of the constant is not used, the constant itself has a meaning :)
Fine for me to use `iota` in future.

Reply:
oh you're right. Makes sense using iota instead thinking again! hahaha
Sorry for the misunderstanding. :D

Reply:
this envs eill be removed after https://github.com/kiegroup/kogito-cloud-operator/pull/347
or a new jira will be opened for it?

Reply:
We need to merge this first and then @xiezhang7 would rebase his PR to remove them to add the new feature based on configMap. Does not make sense removing here.